app/buyers/views/buyers.py

```diff
@@ -307,6 +308,9 @@ def prepared_response_contents_for_brief(brief, responses):
     ess_req_names = brief.get(ESS, [])
     nth_req_names = brief.get(NTH, [])
 
+    def csv_cell_sanitize(text):
+        return re.sub(r"^(=|\+|-|@|!|\|)+", '', text.strip())
```

Comment:
does xlsxwriter have some existing functions for doing this? might be more robust to use that than a random regex